src/cnv/ugbio_cnv/combine_cnmops_cnvpytor_cnv_calls.py

```diff
@@ -0,0 +1,288 @@
+import argparse
+import logging
+import subprocess
+import sys
+from os.path import join as pjoin
+
+import pandas as pd
+import ugbio_cnv.convert_combined_cnv_results_to_vcf
+from ugbio_core.logger import logger
+
+
+def run_cmd(cmd):
+    logger.info(cmd)
+    subprocess.run(cmd, shell=True, check=True)  # noqa: S602
+
+
+def __parse_args(argv: list[str]) -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        prog="cnv_results_to_vcf.py", description="converts CNV calls in bed format to vcf."
+    )
+
+    parser.add_argument("--cnmops_cnv_calls", help="input bed file holding cn.mops CNV calls", required=True, type=str)
+    parser.add_argument(
+        "--cnvpytor_cnv_calls", help="input bed file holding cnvpytor CNV calls", required=True, type=str
+    )
+    parser.add_argument(
+        "--DEL_jalign_merged_results", help="jaign results for Deletions in tsv format", required=True, type=str
+    )
+    parser.add_argument(
+        "--deletions_length_cutoff", help="deletions length cutoff", required=False, type=int, default=3000
+    )
+    parser.add_argument(
+        "--jalign_written_cutoff",
+        help="minimal number of supporting jaligned reads for DEL",
+        required=False,
+        type=int,
+        default=1,
+    )
+    parser.add_argument(
+        "--distance_threshold",
+        help="distance threshold for merging CNV segments",
+        required=False,
+        type=int,
+        default=1500,
+    )
+    parser.add_argument(
+        "--duplication_length_cutoff_for_cnmops_filter",
+        help="duplication_length_cutoff_for_cnmops_filter",
+        required=False,
+        type=int,
+        default=10000,
+    )
+    parser.add_argument("--ug_cnv_lcr", help="UG-CNV-LCR bed file", required=True, type=str)
+    parser.add_argument("--fasta_index", help="fasta.fai file", required=True, type=str)
+
+    parser.add_argument("--out_directory", help="output directory", required=False, type=str)
+    parser.add_argument("--sample_name", help="sample name", required=True, type=str)
+    parser.add_argument("--verbosity", help="Verbosity: ERROR, WARNING, INFO, DEBUG", required=False, default="INFO")
+
+    return parser.parse_args(argv[1:])
+
+
+def get_cnmops_cnv_calls(cnmops_cnv_calls, sample_name, out_directory):
+    cnmops_cnvs_DUP = pjoin(out_directory, f"{sample_name}.cnmops_cnvs.DUP.bed")  # noqa: N806
```

Comment:
why not rename to `cnmops_cnvs_dup` and prevent the need to add the` # noqa `? 
same to all `df_cnmops_cnvs_DUP` below